distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1186,8 +1186,8 @@ type Table
           name is used.
     @column Widget_Helpers.make_column_name_selector
     expand_column : Text | Integer -> Vector | Nothing -> Text | Table -> Table ! Type_Error | No_Such_Column | Index_Out_Of_Bounds
-    expand_column self (column : Text | Integer) (fields : Vector | Nothing = Nothing) (prefix : Text | Nothing = Nothing) =
-        Expand_Objects_Helpers.expand_column self column fields prefix
+    expand_column self (column : Text | Integer) (fields : Vector | Nothing = Nothing) (prefix : Text | Nothing = Nothing) (omit_prefix : Boolean = False) =
+        Expand_Objects_Helpers.expand_column self column fields prefix omit_prefix
```

Comment:
we had this option by having empty string as the prefix to omit it.

Maybe an atom type would clearer:
```
type Prefix_Name
     None
     Column_Name
     Custom value:Text
```

Reply:
Have a `Prefix_Name.from (that:Text)` and then if a user enters text it just works.

Reply:
Done